src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -336,6 +336,12 @@ function MoneyRequestView({
     const noticeTypeViolations = transactionViolations?.filter((violation) => violation.type === 'notice').map((v) => ViolationsUtils.getViolationTranslation(v, translate)) ?? [];
     const shouldShowNotesViolations = !isReceiptBeingScanned && canUseViolations && ReportUtils.isPaidGroupPolicy(report);
 
+    const errors = {...(transaction?.errors ?? {})};
+    if (parentReportAction?.errors) {
+        // Merging parentReportAction.errors into errors
+        Object.assign(errors, parentReportAction.errors);
```

Comment:
@kaushiktd you could combine this assignment in the 339 assignment as well.

Kindly also fix the merge conflicts

Reply:
@abdulrahuman5196 Please check now.

Reply:
Any update @abdulrahuman5196 ?

Reply:
Any update here, @abdulrahuman5196 ?